quickfixj-core/src/test/java/quickfix/SocketInitiatorTest.java

```diff
@@ -145,7 +145,7 @@ public void testBlockLogoffAfterLogon() throws Exception {
                 clientApplication.stopAfterLogon(initiator);
                 clientApplication.setUpLogonExpectation();
 
-                initiator.block();
+                initiator.start();
```

Comment:
I have reverted this particular line, later on you can decide if to keep the test or remove it completely along with the block method.